src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -368,6 +388,12 @@ case class TagFamilySizeMetric(family_size: Int,
     |
     |`edit`, `adjacency` and `paired` make use of the `--edits` parameter to control the matching of
     |non-identical UMIs.
+    |
+    |If the `--min-umi-length` is not given, then all UMIs should have the same length.  If given, then reads that have
```

Comment:
Suggested wording:

By default, all UMIs must be the same length.  If `--min-umi-length=len` is specified then reads that have a UMI shorter than `len` will be discarded, and when comparing UMIs of different lengths, the first `len` bases will be compared, where `len` is the length of the shortest UMI.  The UMI length is the number of `[ACGT]` bases in the UMI (i.e. does not count dashes and other non-ACGT characters).  For reads with UMI pairs (i.e. using the paired assigner), the length is the combined length of the UMIs for the given read (ex. "AAAA-GGGG" has length eight).